src/backend/src/prisma/schema.prisma

```diff
@@ -523,8 +523,10 @@ model Material_Type {
 }
 
 model Manufacturer {
-  name        String     @id
-  dateCreated DateTime
-  creatorId   Int
-  materials   Material[]
+  name            String     @id
+  dateCreated     DateTime
+  creatorId       Int
+  dateDeleted     DateTime?
+  deletedByUserId Int?
```

Comment:
We should also have a field for the actual user - a relation to User using this user id